### DISCUSSION TITLE
Note that 'connect-src' was not explicitly set

### DISCUSSION THREAD

**Discussion by Eshaftri** [directus/directus#14684] [Archive]
I have a frontend app in Vuejs and directus run as a backend server, each running in a different domain. when the user tries login from the frontend using API.I'm getting  this error:

<img width="1656" alt="Screenshot 2022-07-27 at 4 22 55 AM" src="https://user-images.githubusercontent.com/21362631/181151482-2323604e-4ffb-4203-a3a0-2c7f7e83f689.png">

However, this only happens in production,

**Comment by br41nslug** [comment]
You can extend the content security policy using `CONTENT_SECURITY_POLICY_*` environment variables https://docs.directus.io/self-hosted/config-options/#security

**Comment by rijkvanzanten** [comment]
Could also be that your production Vue app has CSP headers that are preventing the Directus API from being accessed 🤔
